pilot/pkg/leaderelection/k8sleaderelection/leaderelection.go

```diff
@@ -233,20 +233,6 @@ func (le *LeaderElector) Run(ctx context.Context) {
 	le.renew(ctx)
 }
 
-// RunOrDie starts a client with the provided config or panics if the config
```

Comment:
I think it is used in istio-cni

Reply:
istio-cni used the k8s client-go leaderelection package
```
import (
	"k8s.io/client-go/tools/leaderelection"
	"k8s.io/client-go/tools/leaderelection/resourcelock"
)
leaderelection.RunOrDie(ctx, leaderelection.LeaderElectionConfig{
	Lock:            lock,
	ReleaseOnCancel: true,
	LeaseDuration:   60 * time.Second,
	RenewDeadline:   15 * time.Second,
	RetryPeriod:     5 * time.Second,
	Callbacks:       leadelectionCallback,
})
```